applications/app/views/fragments/paidAdvertisementGalleryBody.scala.html

```diff
@@ -75,7 +75,11 @@
 
     @if(!page.gallery.content.shouldHideAdverts && rowNum % adInterval == 0) {
         <li class="gallery2__item gallery2__item--advert">
-            <div class="gallery2__img-container js-gallery-slot">
+            <div class="gallery2__img-container">
+                @defining(rowNum / adInterval) { inlineId =>
+                    @gallerySlot(inlineId)
+                    @gallerySlot(inlineId - 1, isMobile = true)
+                }
             </div>
         </li>
     }
```

Comment:
I remember it was forked because the theming of "paid for" content did not play well with the new gallery layout. @blongden73 is supposed to make a round of design update to glabs templates so that they better align across platforms ... it might be a good opportunity to get rid of that cop out.